### DISCUSSION TITLE
reverse proxy example

### DISCUSSION THREAD

**Discussion by superdolt** [vlang/v#12707] [Ideas]
really hope for a reverse proxy example on pico.v

**Comment by ghost** [comment]
what do you mean?
`picoev.v` doesn't seem to have the ability to be a  proxy server..
If you mean that you want to use `pico.v` as a backend server, you can just set up a reverse proxy settings to http-server like Nginx or h2o..

**Comment by superdolt** [comment]
Using pico.v as the reverse proxy by itself. How to do this?

**Comment by ghost** [comment]
no examples since `pico.v` is not a reverse proxy server.

**Comment by superdolt** [comment]
How do I do reverse proxy with pico?

**Comment by ghost** [comment]
If you want to implement reverse proxy server with picoev.v in V, you have to implement it in V yourself, referring to for instance https://github.com/h2o/h2o. which is the successor project of pico.

**Comment by vcker** [comment]
https://github.com/vlang/v/discussions/21428
i am write a reverse proxy, but have a proplem, help me.
